### ISSUE TITLE
ember-data.js.map isn't being updated on cdnjs.com

### ISSUE THREAD

**Issue by gerry3** [components/ember-data#28]
`ember-data.js.map` isn't being updated with the other `ember-data.js` files on <https://cdnjs.com>

<https://cdnjs.cloudflare.com/ajax/libs/ember-data.js/1.0.0-beta.18/ember-data.js.map> is actually for 1.0.0-beta.16.1, not 1.0.0-beta.18. Similarly, 1.0.0-beta.17 is also using the map from 1.0.0-beta.16.1.

I opened cdnjs/cdnjs#4692, but it was closed as an issue from "upstream". Is this the right place?

Thanks for any help.

@fivetanley

**Comment by rwjblue** [comment]
the map file is here, not sure what they mean by upstream

**Comment by rwjblue** [comment]
ahh you are saying it is the wrong version?

**Comment by gerry3** [comment]
@rwjblue correct. it seems to be pulling `1.0.0-beta.16.1` for all later version so far (17 & 18)

**Comment by rwjblue** [comment]
Yeah, we need to update the file and re-tag.

**Comment by gerry3** [comment]
It would be nice to fix the old ones. Is re-tagging possible? Can I help?

But even more important, we need to update the process so that future tags include the correct map file. I guess that piece isn't automated? Can it be? Or could we add a test or hook that breaks if this file isn't up-to-date?
